.drone.star

```diff
@@ -2799,3 +2771,135 @@ def setupForLitmus():
             "cat .env",
         ],
     }]
+
+def performWebCache(ctx):
+    ocis_git_base_url = "https://raw.githubusercontent.com/owncloud/ocis"
+    path_to_drone_env = "%s/%s/.drone.env" % (ocis_git_base_url, ctx.build.commit)
+    path_to_check_script = "%s/%s/tests/config/drone/check_web_cache.sh" % (ocis_git_base_url, ctx.build.commit)
+
+    return [
+        {
+            "name": "get-drone-env-and-check-script",
+            "image": OC_UBUNTU,
+            "commands": [
+                "curl -s -o .drone.env %s" % path_to_drone_env,
+                "curl -s -o check_web_cache.sh %s" % path_to_check_script,
+                "chmod +x check_web_cache.sh",
```

Comment:
I don't think you need this because you are using `bash check_web_cache.sh`, the executable is `bash` in that case